piker/brokers/cli.py

```diff
@@ -0,0 +1,61 @@
+"""
+Console interface to broker client/daemons.
+"""
+import json
+from functools import partial
+from importlib import import_module
+
+import click
+import trio
+
+from ..log import get_console_log, colorize_json
+
+
+def run(main, loglevel='info'):
+    log = get_console_log(loglevel)
+
+    # main sandwich
+    try:
+        return trio.run(main)
+    except Exception as err:
+        log.exception(err)
+    finally:
+        log.debug("Exiting piker")
```

Comment:
I'm not crazy in that this code isn't being used yet?

Reply:
This one, the main function here.

Reply:
It's used on lines `32` and `61`?

Reply:
Derp